README.md

```diff
@@ -10,36 +10,29 @@ python3.8
 python3-pip
 python3-venv
 manim
-make
 git # of course
 ```
 
-Please, use the python from virtual environment instead of the global python.
-First, create the virtual environment by simply running `make` command in this directory.
-
-Each time you open a terminal, run the following in this directory. (TODO: update for other OS, where this doesn't work)
-```
-. .venv/current/bin/activate
-```
-
-That's it :)
-
 ### Create the video
 
 FOR LINUX, MACOS: 
 ```
-LANGUAGE=armenian PYTHONPATH=lib:objects python qarakusi.am.py 10674
+PYTHONPATH=lib:objects python qarakusi.am.py 10674 --language armenian -pqh
```

Comment:
Perhaps instead of python, python3?

Reply:
Let's not do that. Everybody knows their computer better than we do. You might as well say "python3.8 instead of python3".